fe/fe-core/src/main/java/org/apache/doris/nereids/rules/implementation/AggregateStrategies.java

```diff
@@ -2076,7 +2079,8 @@ private boolean couldConvertToMulti(LogicalAggregate<? extends Plan> aggregate)
             if (!func.isDistinct()) {
                 continue;
             }
-            if (!(func instanceof Count || func instanceof Sum || func instanceof GroupConcat)) {
+            if (!(func instanceof Count || func instanceof Sum || func instanceof GroupConcat
```

Comment:
how about add a new Interface "MultiDistinct", and let these Agg functions implement it?

Reply:
Yes, an interface is added in #45209, I will use that interface  in that pr.